cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,51 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
-    
+  it('Login with valid creds', () => {
+    cy.visit('/login');
+
+    cy.get('#userName').type('nkond.test');
+    cy.get('#password').type('Passwprd!123');
+    cy.contains('.text-right button', 'Login').click();
+
+    cy.get('#userName-value').should('contain.text', 'nkond.test');
+    cy.assertPageUrl('/profile');
   });
 
-  it('', () => {
```

Comment:
Provide a meaningful description for each test case. Replace the empty string with a description.